### PR TITLE
fix(ui): missing manifests and logs (#9231)

### LINKED ISSUE #9231: ArgoCD: Logs tab is not visible in latest version of ArgoCD
Checklist:

* [Yes ] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [Yes ] I've included steps to reproduce the bug.
* [Yes ] I've pasted the output of `argocd version`.

**Describe the bug**

Logs tab is not visible which used to come before. This is broken with latest tag of ArgoCD

**To Reproduce**

Click on the pod kind and observe the pop-up.

**Expected behavior**

Logs tab should be visible for Pods.

**Screenshots**

ArgoCD

**Version**

```
{
    "Version": "v2.4.0+5f0e58d",
    "BuildDate": "2022-04-27T17:34:43Z",
    "GitCommit": "5f0e58d5f0678a0ca545c82a0f0ed060fdf648d3",
    "GitTreeState": "clean",
    "GoVersion": "go1.18.1",
    "Compiler": "gc",
    "Platform": "linux/amd64",
    "KustomizeVersion": "v4.4.1 2021-11-11T23:36:27Z",
    "HelmVersion": "v3.8.1+g5cb9af4",
    "KubectlVersion": "v0.23.1",
    "JsonnetVersion": "v0.18.0"
}
```

**Logs**

```
NA
```

### PR DESCRIPTION
**Pull request by crenshaw-dev**
Fixes #9231

### CODE REVIEW

File: ui/src/app/shared/services/applications-service.ts [hunk 1]
```diff
@@ -355,9 +355,7 @@ export class ApplicationsService {
         const search = new URLSearchParams();
         search.set('container', containerName);
         search.set('namespace', namespace);
-        if (follow) {
-            search.set('follow', follow.toString());
-        }
+        search.set('follow', follow.toString());
```
Comment (pasha-codefresh) [inline ui/src/app/shared/services/applications-service.ts:358]: why this change is required? actually with this i think you will not be able to disable follow mode, no ?

Comment (crenshaw-dev) [reply ui/src/app/shared/services/applications-service.ts:358]: After a recent [grpc upgrade](https://github.com/argoproj/argo-cd/pull/8929) some previously-optional fields became required by the API. I haven't dug into precisely why they became required.

I don't think this change will make it impossible to disable follow mode. It just ensures that whatever `follow` value is set (true or false), it's encoded in the query string. I'll test locally to be sure.

Comment (crenshaw-dev) [reply ui/src/app/shared/services/applications-service.ts:358]: @pasha-codefresh I've confirmed that follow/non-follow still works. I've also pushed a couple more `group || ''` changes. I discovered.

**Review by pasha-codefresh — APPROVED** [review]
Great job

**Pull request merged**